site/content/en/docs/Getting started/installation.md

```diff
@@ -4,94 +4,102 @@ linkTitle: "Install Kustomize"
 
-## Packages
-Kustomize is also available in some package repositories.
+The following [script] detects your OS and downloads the appropriate kustomize binary to your
+current working directory.
 
-### Debian/Ubuntu
 ```bash
-sudo apt-get install kustomize
+curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
 ```
-### Arch
+
+**This script doesn't work for ARM architecture.** If you want to install ARM binaries, please
+go to the release page to find the URL.
+
+## Docker Images
+
+Starting with Kustomize v3.8.7, docker images are available to run Kustomize.
+The images are hosted in kubernetes official GCR repositories.
+
+See [GCR page] for available images.
+
+The following commands are how to pull and run kustomize 3.8.7 docker image.
+
 ```bash
-pacman -S kustomize
+docker pull k8s.gcr.io/kustomize/kustomize:v3.8.7
+docker run k8s.gcr.io/kustomize/kustomize:v3.8.7 version
 ```
 
-### Mac
-[Homebrew](https://brew.sh/):
+## Homebrew / MacPorts
+
+For [Homebrew] users:
+
 ```bash
 brew install kustomize
 ```
 
-[MacPorts](https://www.macports.org/):
-```bash
-sudo port install kustomize
-```
+For [MacPorts] users:
 
-### Windows
-[Chocolatey](https://community.chocolatey.org/packages/kustomize)
 ```bash
-choco install kustomize
+sudo port install kustomize
 ```
 
-## Docker
-Docker images for kustomize are published on the [GCR Container Registry](https://console.cloud.google.com/gcr/images/k8s-artifacts-prod/US/kustomize/kustomize).
-```bash
-docker run k8s.gcr.io/kustomize/kustomize:v4.5.1 version
-```
+## Chocolatey
 
-## go get
-<!--
-TODO: is this still the way to do this? v3 seems old and my go module knowledge isn't great
--->
-Requires [Go](https://go.dev/) to be installed.
 ```bash
-GOBIN=$(pwd)/ GO111MODULE=on go get sigs.k8s.io/kustomize/kustomize/v3
+choco install kustomize
 ```
 
-## Source
-<!--
-TODO: once again here, are these instructions up-to-date? We probably should bump to a later version of kustomize but I'm also not sure if these go env variables are modern practice
--->
-Clone the kustomize Github repo and build using go.
-```bash
-# Need go 1.13 or higher
-unset GOPATH
-# see https://golang.org/doc/go1.13#modules
-unset GO111MODULES
-
-# clone the repo
-git clone git@github.com:kubernetes-sigs/kustomize.git
-# get into the repo root
-cd kustomize
-
-# Optionally checkout a particular tag if you don't
-# want to build at head
-git checkout kustomize/v3.2.3
+For support on the chocolatey package
+and prior releases, see:
 
-# build the binary
-(cd kustomize; go install .)
+- [Choco Package]
+- [Package Source]
 
-# run it
-~/go/bin/kustomize version
\ No newline at end of file
+[Go]: https://golang.org
+[releases page]: https://github.com/kubernetes-sigs/kustomize/releases
+[script]: https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh
+[GCR page]: https://us.gcr.io/k8s-artifacts-prod/kustomize/kustomize
```

Comment:
nit: folks probably don't know what GCR stands for, so maybe something like "Kustomize's artifacts on Google Container Registry" would be more helpful.

Reply:
I added the GCR explanation under the "Docker Images" section